channel/DI.cc

```diff
@@ -71,3 +114,19 @@ void DI::Reset() {
         resets |= 0x400;
     }
 }
+
+void DI::MakeCommand(u32 command, u32 sub0, u32 sub1) {
```

Comment:
`u8`, `u8`, `u16` so we can remove the masks?

Reply:
I preferred mask validation within the function as opposed to handling it at the call because I forgot this is private and only callable in C++ contexts. Fixing.